samwise.js

```diff
@@ -1,44 +1,47 @@
-/**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?